tests/ui/linkage-attr/raw-dylib/elf/basic-symbol-links-with.rs

```diff
@@ -0,0 +1,19 @@
+//@ only-elf
+
+//@ build-pass
+
+//! Ensures that linking succeeds with raw-dylib when the linked against
+//! library is not available.
+
+#![feature(raw_dylib_elf)]
+#![allow(incomplete_features)]
+
+#[link(name = "rawdylibbutforcats", kind = "raw-dylib")]
```

Comment:
Maybe merge this test and ui/linkage-attr/raw-dylib/elf/basic-symbol-error-without.rs by using multiple revisions?